deploy/fat_container/templates/docker.env.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -o nounset
+
+MONGO_PASSWORD="$1"
+ENCRYPTION_PASSWORD="$2"
+ENCRYPTION_SALT="$3"
+
+cat <<EOF
+# Sentry
+APPSMITH_SENTRY_DSN=
+
+# Smart look
+APPSMITH_SMART_LOOK_ID=
+
+# Google OAuth
+APPSMITH_OAUTH2_GOOGLE_CLIENT_ID=
+APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET=
+
+# Github OAuth
+APPSMITH_OAUTH2_GITHUB_CLIENT_ID=
+APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET=
+
+# Segment
+APPSMITH_SEGMENT_KEY=
+
+# RapidAPI
+APPSMITH_RAPID_API_KEY_VALUE=
+APPSMITH_MARKETPLACE_ENABLED=
+
+# Optimizely
+APPSMITH_OPTIMIZELY_KEY=
+
+# Algolia Search (Docs)
+APPSMITH_ALGOLIA_API_ID=
+APPSMITH_ALGOLIA_API_KEY=
+APPSMITH_ALGOLIA_SEARCH_INDEX_NAME=
+
+#Client log level (debug | error)
+APPSMITH_CLIENT_LOG_LEVEL=
+
+# GOOGLE client API KEY
+APPSMITH_GOOGLE_MAPS_API_KEY=
+
+# Email server
+APPSMITH_MAIL_ENABLED=
+APPSMITH_MAIL_HOST=
+APPSMITH_MAIL_PORT=
+APPSMITH_MAIL_USERNAME=
+APPSMITH_MAIL_PASSWORD=
+
+# Email server feature toggles
+# true | false values
+APPSMITH_MAIL_SMTP_AUTH=
+APPSMITH_MAIL_SMTP_TLS_ENABLED=
+
+# Disable all telemetry
+# Note: This only takes effect in self-hosted scenarios. 
+# Please visit: https://docs.appsmith.com/v/v1.2.1/setup/telemetry to read more about anonymized data collected by Appsmith
+# APPSMITH_DISABLE_TELEMETRY=false
```

Comment:
Let's uncomment env variables, where the value is a valid value (instead of an empty string). We've seen in the past where folks would change the value of an env variable, but forget to uncomment, and then come complaining on Discord that it doesn't work. Since the value `false` here is a valid value here, let's uncomment this proactively.

```suggestion
APPSMITH_DISABLE_TELEMETRY=false
```